imagededup/utils/plotter.py

```diff
@@ -0,0 +1,132 @@
+import matplotlib.gridspec as gridspec
```

Comment:
matplotlib is missing from dependencies in `setup.py`

Reply:
tqdm and sklearn as well

Reply:
Added the missing dependencies.